### ISSUE TITLE
Seems translation to Russian for "Congratulations. You have successfully completed this lesson."  phrase is broken.

### ISSUE THREAD

**Issue by rusboy** [WebGoat/WebGoat#199]
I'm observing ???? letters instead phrase "Congratulations. You have successfully completed this lesson."
Look at picture in the issue below
Original issue https://github.com/WebGoat/WebGoat-Lessons/issues/58

**Comment by rusboy** [comment]
Pardon, wrong button was pressed

**Comment by rusboy** [comment]
Hey Guys,
Just wanted to take a look solution for the issue.. But.. Seems it has been already fixed by https://github.com/WebGoat/WebGoat/commit/daa05dd19278497ba8349574fc79bc65de4a56c9 commit.

I've just done some tests.. I tried to compile package based on more older code. Issue occurs. Begin from the commit, there is no the issue.

As well, I tested on latest develop branch. There is no the issue as well.

Seems, we can close this issue request.

Thanks
